runtime/compiler/env/VMJ9.cpp

```diff
@@ -4220,7 +4221,7 @@ TR_J9VMBase::initializeLocalObjectFlags(TR::Compilation * comp, TR::Node * alloc
    int32_t initValue = TR::Compiler->cls.romClassOf(ramClass)->instanceShape;
 #endif
 
-   if (!TR::Options::getCmdLineOptions()->realTimeGC())
+   if (!comp->getOptions()->realTimeGC())
```

Comment:
This is problematic. We are looking at the thread on the server. I think we need to send a message for this one, but definitely needs more thought. It is the default behavior though.

Reply:
I think `TR_J9VMBase::initializeLocalObjectFlags()` should be overridden  in `TR_J9ServerVM` to retrieve the `initValue`.